js_modules/dagster-ui/packages/ui-core/src/assets/useAssetDefinitionFilterState.oss.tsx

```diff
@@ -224,3 +224,22 @@ export function filterAssetDefinition(
 
   return true;
 }
+
+export function getAssetFilterStateQueryString() {
+  const values: Record<keyof AssetFilterType, '1'> = {
+    groups: '1',
+    kinds: '1',
+    changedInBranch: '1',
+    owners: '1',
+    tags: '1',
+    codeLocations: '1',
+    selectAllFilters: '1',
+  };
```

Comment:
Could define this outside the function since it's static.

Reply:
👍